Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6-onbuild
```

Comment:
I know this is a bit uglier but all it has is alpine and the binary when its done https://github.com/cloudflare/cfssl/blob/master/Dockerfile.minimal

its the same route i take in all my personal go projects as well https://github.com/jfrazelle/weather/blob/master/Dockerfile

(: